pkgs/by-name/pa/papertrail/package.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation {
     ln -s ${papertrail-env}/bin/papertrail $out/bin/papertrail
   '';
 
-  passthru.updateScript = bundlerUpdateScript "papertrail";
+  passthru.updateScript = bundlerUpdateScript pname;
```

Comment:
:-1: https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Can you elaborate on this?